src/fr/composants/boite/case-dusage.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Boîte
+layout: "layouts/component-documentation.njk"
+eleventyNavigation:
+  key: containerFR
+  title: Boîte
+  locale: fr
+  parent: componentsFR
+  otherNames: conteneur, encadré.
+  description: Il s’agit d’un modèle de boîte simple associé à une largeur de contenu fixe.
+  thumbnail: /images/common/components/preview-container.svg
+  alt: Un rectangle avec une bordure entourant des rectangles gris représentant le contenu.
```

Comment:
Can you set the state to `published`? The component is currently not showing up under our published components.